ansible/www-standalone/tools/promote/upload_to_cloudflare.sh

```diff
@@ -31,9 +31,10 @@ if [ -z ${staging_bucket+x} ]; then
   exit 1
 fi
 
-relativedir=${dstdir/$dist_rootdir/"$site/"}
+relative_srcdir=${srcdir/$staging_rootdir/"$site/"}
+relative_dstdir=${dstdir/$dist_rootdir/"$site/"}
 tmpversion=$2
 
-rclone copy $staging_bucket/$relativedir/$tmpversion/ $prod_bucket/$relativedir/$tmpversion/
-rclone copyto $staging_bucket/$relativedir/index.json $prod_bucket/$relativedir/index.json
-rclone copyto $staging_bucket/$relativedir/index.tab $prod_bucket/$relativedir/index.tab
+rclone copy $staging_bucket/$relative_srcdir/$tmpversion/ $prod_bucket/$relative_dstdir/$tmpversion/
+rclone copyto $staging_bucket/$relative_srcdir/index.json $prod_bucket/$relative_dstdir/index.json
+rclone copyto $staging_bucket/$relative_srcdir/index.tab $prod_bucket/$relative_dstdir/index.tab
```

Comment:
I think we should use `$relative_dstdir` as the source for the index files. 
They are copied to the right folder in https://github.com/nodejs/build/blob/8fc01e083c2382bf86cf0df395dbf46748b55656/ansible/www-standalone/tools/promote/_resha.sh#L41-L49